TwilioVerifyDemo/TwilioVerifyDemo/TwilioVerify/TwilioVerifyAdapter.swift

```diff
@@ -18,6 +18,10 @@
 import Foundation
 import TwilioVerifySDK
 
+private enum Constants {
+  static var accessGroup: String? { KeyChainAccessGroupHelper.getAccessGroupInfo()?.rawValue }
```

Comment:
Should this be read from the [project configuration](https://github.com/twilio/twilio-verify-ios/pull/180/files#diff-f7e1ba16e27d55e50e31ecc2628915ff4942a40d60e17eaebee43aa13c5cf679R11) instead of a [stored value](https://github.com/twilio/twilio-verify-ios/pull/180/files#diff-01970e6a9b70d7f99d9fc8c173288201c5524a57c32d88b151e6375107aaeabfR20)?

Reply:
Using this helper is the only way to read that entitlement file, without writing down the TeamID.

Reply:
What is the best practice for this? Are other libraries using a helper to get the access group?